Stdin/README.md

```diff
@@ -0,0 +1,141 @@
+# Stdin Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Stdin     | 8796536366ee21852db56dccb60bc564598b618c865fc50c8b1ab740bba128e3  |
+
+
+## Description
+
+Allows to get input from user for basic types like integers, addresses, public keys, strings, cells.
+
+## Functions
+
+Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
+
+`inputStr` - prints prompt message to the user and returns string entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: bytes - string entered by user.
+
+`inputInt` - prints prompt message to the user and returns singed integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: int256 - integer entered by user
+
+`inputUint` - prints prompt message to the user and returns unsinged integer entered by user
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+	
+	prompt: bytes - string printed to the user and describing what to enter
+
+returns: 
+
+	value: uint256 - integer entered by user
+
+`inputAddr` - prints prompt message to the user and returns address entered by user
```

Comment:
I don't think we need this input in std, it's better to have an independent interface for such case (it will also be implemented independently in front-end, as it looks differently from the regular "chat input")